### ISSUE TITLE
Keep worker going in background?

### ISSUE THREAD

**Issue by DarrylD** [devfd/react-native-workers#5]
In the readme for lifecycle:
>the workers are paused when the app enters in the background

In the description it's:
> Bonus point on Android you can keep a worker alive even when a user quit the application 🎆

How should I keep it from pausing?

**Comment by DarrylD** [comment]
@devfd any idea on this before my attempt (java is not my strong suit) at forking?

**Comment by devfd** [comment]
yeah sure. actually there is already some code implemented. just need to be freshen-up with latest RN version. it should be coming shortly.

**Comment by anhnt** [comment]
hi @devfd can you please give an ETA when this feature is ready to ship

**Comment by DarrylD** [comment]
kinda looking forward this too as it's one of the most needed plugin for react native. Willing to help if needed :)

**Comment by freerider7777** [comment]
Also waiting :)

**Comment by sudharsan1988** [comment]
+1

**Comment by DorWeid** [comment]
+1

**Comment by jasiek-net** [comment]
+1

**Comment by vicmpen** [comment]
+1

**Comment by jsdevel** [comment]
ping @devfd

**Comment by curtiszimmerman** [comment]
+1

**Comment by brafdlog** [comment]
+1

**Comment by rexlow** [comment]
+1

**Comment by realart** [comment]
+1.

**Comment by piranna** [comment]
Any update on this? Seems like the project is abandoned... :-/

**Comment by Samykills** [comment]
yeah it is kinda abandoned :(
